easybuild/easyconfigs/n/NetLogo/NetLogo-5.3.1-64.eb

```diff
@@ -0,0 +1,25 @@
+easyblock = 'PackedBinary'
+
+name = 'NetLogo'
+version = '5.3.1'
+versionsuffix = '-64'
+
+homepage = 'http://ccl.northwestern.edu/netlogo/'
+description = """NetLogo is a multi-agent programmable modeling environment. It
+is used by tens of thousands of students, teachers and researchers worldwide.
+It also powers HubNet participatory simulations. It is authored by Uri Wilensky
+and developed at the CCL."""
+
+# NetLogo is precompiled and needs no particular toolchain.
+# It even comes with its own Java.
+toolchain = {'name': 'dummy', 'version': 'dummy'}
+
+source_urls = ['http://ccl.northwestern.edu/netlogo/%(version)s/']
+sources = ['%(name)s-%(version)s%(versionsuffix)s.tgz']
+
+sanity_check_paths = {
+    'files': ['NetLogo', 'NetLogo3D', 'NetLogoLogging'],
```

Comment:
Probably not a bad idea, but I'd ordinarily use `prepend-path` (in the module file) for that purpose. Can `modextravars` be used to create a `prepend-path` statement?